src/cryptonote_core/miner.h

```diff
@@ -35,7 +35,19 @@
 #include "cryptonote_basic.h"
 #include "difficulty.h"
 #include "math_helper.h"
+//#include <boost/circular_buffer.hpp>
 
+#define BACKGROUND_MINING_DEFAULT_IDLE_THRESHOLD_PERCENTAGE       90
```

Comment:
Can these be `constexpr const` values instead? I don't see an advantage to using the c-preprocessor in this context.